### PR TITLE
[MU4] Add tremolo section for Inspector

### PR DESCRIPTION
**Pull request by Howard-C**
Icons made by Tantacrul.

image

Currently the third option is not working because this stroke style hasn't been implemented yet.

Co-Authored-By: Tantacrul <tantacrul@users.noreply.github.com>

### CODE REVIEW

**Comment by Howard-C** [comment]
To team members: I don't know whether I handled it correctly with the font update (can it be committed by me, even though I added @Tantacrul as a co-author?), so please tell me if it's not.

**Comment by Howard-C** [comment]
To team members: I don't know whether I handled it correctly with the font update (can it be committed by me, even though I added @Tantacrul as a co-author?), so please tell me if I didn't.

File: mu4/scenes/inspector/internal/services/elementrepositoryservice.cpp [hunk 1]
```diff
@@ -323,3 +329,20 @@ QList<Ms::Element*> ElementRepositoryService::findTexts() const
 
     return resultList;
 }
+
+QList<Ms::Element*> ElementRepositoryService::findTremolos() const
+{
+    QList<Ms::Element*> resultList;
+
+    for (Ms::Element* element : m_elementList) {
+        Ms::Tremolo* tremolo = Ms::toTremolo(element);
```
Comment (mattmcclinch) [inline mu4/scenes/inspector/internal/services/elementrepositoryservice.cpp:338]: You are casting to a tremolo without first checking the element type. So, of course, `CONVERT(Tremolo)` will trigger an assertion failure every time there are any elements selected which are not tremolos.

Comment (Howard-C) [reply mu4/scenes/inspector/internal/services/elementrepositoryservice.cpp:338]: Oh that was foolish of me, sorry!

**Pull request merged**